01/assets/js/app.js

```diff
@@ -1,4 +1,50 @@
 const a = '4.2';
 const b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b);
+
+// 1. Check the types of variables
+console.log("Type of variable a: ", typeof a); // string
+console.log("Type of variable b: ", typeof b); // number
+
+// 2. Arithmetic operations
+// Convert 'a' to a number to avoid string concatenation
+const addition = parseFloat(a) + b;
+
+const subtraction = a - b;
+const multiplication = a * b;
+const division = a / b;
+const modulo = a % b;
+const exponentiation = a ** b;
+
+// console.log("Addition: ", addition);
+// console.log("Subtraction: ", subtraction);
+// console.log("Multiplication: ", multiplication);
+// console.log("Division: ", division);
+// console.log("Modulo: ", modulo);
+// console.log("Exponentiation: ", exponentiation);
+
+// 3. Check which result is greater than 20, which is less than 20 and which is equal to 20
+const results = [
```

Comment:
Zastanawiam się czy nie lepiej byłoby zrobić z tego funkcję, która przyjmuje przez parametr nazwy oraz funkcji z działaniem. Natomiast 2 parametry to coś stałego, co potem można przekazać. Może dałoby się do tego wykorzystać wzorzec obserwator: https://refactoring.guru/pl/design-patterns/observer